src/dodal/beamlines/i04_1.py

```diff
@@ -123,6 +123,7 @@ def undulator(
         f"{BeamlinePrefix(BL).insertion_prefix}-MO-SERVC-01:",
         wait_for_connection,
         fake_with_ophyd_sim,
+        id_gap_lookup_table_path="SPECIFY_ME",
```

Comment:
I think we should just do https://github.com/DiamondLightSource/dodal/issues/871. I'll do that separate to this PR though